From 58db1d6eaed5d7bc957d02518acdabc891084e09 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 6 Jun 2024 09:32:12 +0200
Subject: [PATCH 0994/1695] dts: stm32mp151: add address-cells/size-cells in
 rcc node

Add #address-cells and #size-cells properties that where missing
in the rcc node for stm32mp15 DT source files.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git b425826eadefc5cee921f2dc3ad01f8215fc444b dts: stm32mp151: add address-cells/size-cells in rcc node]
Change-Id: I27a30923b31d02b732109e3d3e9721d0158670e4
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/386082
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp151.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp151.dtsi b/core/arch/arm/dts/stm32mp151.dtsi
index 305f3a0fa..fccc90d57 100644
--- a/core/arch/arm/dts/stm32mp151.dtsi
+++ b/core/arch/arm/dts/stm32mp151.dtsi
@@ -190,6 +190,8 @@
 			reg = <0x50000000 0x1000>;
 			#clock-cells = <1>;
 			#reset-cells = <1>;
+			#address-cells = <1>;
+			#size-cells = <0>;
 		};
 
 		pwr_regulators: pwr@50001000 {
-- 
2.47.2

